src/integrationTest/groovy/wooga/gradle/macOS/security/tasks/MultiKeychainOperationIntegrationSpec.groovy

```diff
@@ -0,0 +1,69 @@
+package wooga.gradle.macOS.security.tasks
+
+import com.wooga.gradle.test.writers.PropertyGetterTaskWriter
+import com.wooga.gradle.test.writers.PropertySetterWriter
+import spock.lang.Unroll
+import wooga.gradle.macOS.security.SecurityIntegrationSpec
+import wooga.gradle.macOS.security.SecurityMultikeychainOperationSpec
+import wooga.gradle.macOS.security.SecurityTaskIntegrationSpec
+
+import java.lang.reflect.ParameterizedType
+
+import static com.wooga.gradle.test.queries.TestValue.filePath
+import static com.wooga.gradle.test.queries.TestValue.filePaths
+import static com.wooga.gradle.test.writers.PropertySetInvocation.*
+
+abstract class MultiKeychainOperationIntegrationSpec<T extends SecurityMultikeychainOperationSpec> extends SecurityTaskIntegrationSpec<T> {
```

Comment:
Here, you can use implement the TaskIntegrationSpec which adds all these `subject...` property/methods

Reply:
Ah yes. It was already there I forgot to remove the code.